"#1081 Removed public class modifiers"

c-thornton opened the pull request:
<!-- Your content goes here: -->

Removed public class modifiers from the Meetup.java and MeetupSchedule.java src and .meta folders.

<!-- DO NOT EDIT BELOW THIS LINE! -->
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

FridaTveit commented on exercises/meetup/.meta/src/reference/java/Meetup.java:
```diff
@@ -2,7 +2,7 @@
+class Meetup {
     private final LocalDate startOfMonth;
 
     public Meetup(int monthOfYear, int year) {
```
This can also be package private :)

c-thornton replied:
Sorry for missing that one @FridaTveit, thought I got them all. Thanks for the feedback on this. I will get that one modified in a couple hours when I am in front of my computer again and modify the PR.

sjwarner-bp replied:
Perfect!

c-thornton commented:
Thanks @sjwarner-bp for the tip. I was wondering what I was doing incorrectly there. I should have read what @FridaTveit had written better. This is a learning experience for sure. The tests are running again now so hopefully it will pass this time and we should be good.

c-thornton commented:
Doh! I should have known that honestly. Thanks for the link, I will read up on that tonight. I will make the correct change shortly and push it to my fork. Thanks for the guidance @sjwarner-bp and for being patient with me.

sjwarner-bp commented:
That's no problem! Thanks for the speedy updates 👍

c-thornton commented:
So one thing I just thought about @sjwarner-bp was consistency with the style. Since the Meetup method will not have a private declaration should I go ahead and remove it from the cycleToPrev method as well so that the style is consistent within the Meetup class?

sjwarner-bp commented:
That's a good question! While changing the `private` methods to package-private wouldn't cause the build to fail, it is worth keeping in mind that we are trying to [minimise the accessibility of classes and members](http://jtechies.blogspot.com/2012/07/item-13-minimize-accessibility-of.html). 

Because these methods are only used in Meetup (i.e. nothing outside of their class calls them), we can keep them as private.

Remember - package-private is **more** accessible than `private`, and we want to keep accessibility to a (working) minimum 🙂

c-thornton commented:
Awesome, thanks for the link and the info. So I removed private from cycleToPrev and cycleToNext with the commit that I just pushed on my side. I added them back in a pushed again and the checks just finished up for this one and were succesful. Thanks for the extra guidance and patience while working with me on this pull request.

c-thornton commented:
Awesome, thanks for the link and the info. So I removed private from cycleToPrev and cycleToNext with the commit that I just pushed on my side. I added them back in a pushed again and the checks just finished up for this one and were succesful. Thanks for the extra guidance and patience while working with me on this pull request.

c-thornton commented:
Awesome! Thanks @sjwarner-bp. Now time to do some reading before tackling some other issues. Thanks for all the feedback.
